matrix-neoboard-widget/tsconfig.json

```diff
@@ -2,7 +2,16 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "rootDir": ".",
-    "noEmit": true
+    "noEmit": true,
+    "types": ["vite/client"],
+    /* Bundler mode */
```

Comment:
Looks like happy talk. Next line says „bundler“, so yes, it's obviously bundler mode :shrug:

Reply:
What do you mean here? :) Unable to parse this :D